testing/integration/tests/gas_calibration_test.rs

```diff
@@ -550,7 +550,9 @@ fn on_scan_cbor_links() {
         let mut iter_obs: HashMap<String, Vec<Obs>> = Default::default();
 
         for event in ret.exec_trace {
-            let ExecutionEvent::GasCharge(charge) = event else { continue };
+            let ExecutionEvent::GasCharge(charge) = event else {
+                continue;
+            };
```

Comment:
all these `let .. else {}` line breaks, are those coming from new formatting rules? they're not a clippy thing as far as I can tell

Reply:
Yes. Rustfmt changed it's rules.

Reply:
Which is unfortunate. I have some nostalgia for `foo() or die "oops";` from my (very brief) perl days.